doc/migrating_to_s3.md

```diff
@@ -18,14 +18,11 @@ aws s3 sync public/system s3://BUCKET_NAME --acl private
 
 ### Update the application settings
 
-Add the `aws-sdk-s3` gem to `Gemfile`
```

Comment:
I figured this line would still be necessary since it's commented out by default.